quinn-proto/src/tests/mod.rs

```diff
@@ -27,11 +26,21 @@ use crate::{
     crypto::rustls::QuicServerConfig,
     frame::FrameStruct,
     transport_parameters::TransportParameters,
+    Duration, Instant,
 };
 mod util;
 use util::*;
 
-#[test]
+#[cfg(all(target_family = "wasm", target_os = "unknown"))]
+use wasm_bindgen_test::wasm_bindgen_test;
+
+// Enable this if you want to run these tests in the browser.
+// Unfortunately it's either-or: Enable this and you can run in the browser, disable to run in nodejs.
+// #[cfg(all(target_family = "wasm", target_os = "unknown"))]
+// wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
+
+#[cfg_attr(all(target_family = "wasm", target_os = "unknown"), wasm_bindgen_test)]
```

Comment:
I think we should find a way to avoid repeating these on every test.

Reply:
Unfortunately I don't know of a good way :/
What works:
- Writing our own proc macro
- Writing our own declarative macro (I think?)
- Splitting the test into 3 modules: one with `pub fn`s, one with `#[test]`s calling the `pub fn`s, and one with `#[wasm_bindgen_test]`s running the `pub fn`s.

I don't like either of these options. Would love some good ideas.

What doesn't work:
- Use only `#[wasm_bindgen_test]` (it doesn't resolve to just `#[test]` on other platforms, which is a shame IMO)

Reply:
> * Use only `#[wasm_bindgen_test]` (it doesn't resolve to just `#[test]` on other platforms, which is a shame IMO)

Have you investigated whether there is an open issue or some discussion of this in the upstream repo? It seems pretty straightforward...

Reply:
Yeah, I looked, there was nothing. Anyhow, [I opened an issue](https://github.com/rustwasm/wasm-bindgen/issues/4139).
Do you think we should block this PR on that issue & a subsequent release of wasm-bindgen-test?

Reply:
Let's see if there's any movement on your issue until Monday or something? Looks like at least one of the active maintainers in that repo is also a Quinn contributor (👋 @daxpedda), so let's see what happens.

Reply:
Thank you for the ping djc!
https://github.com/rustwasm/wasm-bindgen/pull/4150, part of the latest release, should resolve this issue.
If not please let me know!

Reply:
Not sure why I didn't think of this, but this works, too, at least in our case where we don't need async tests:
```rs
#[cfg(all(target_family = "wasm", target_os = "unknown"))]
use wasm_bindgen_test::wasm_bindgen_test as test;
```

I've changed the PR to use that. The diff is much smaller this way.